tests/functional/spec/features/routing/checkbox_count_routing.spec.js

```diff
@@ -0,0 +1,39 @@
+import ToppingCheckboxPage from "../../../generated_pages/new_routing_checkbox_count/topping-checkbox.page.js";
+import CorrectAnswerPage from "../../../generated_pages/new_routing_checkbox_count/correct-answer.page";
+import IncorrectAnswerPage from "../../../generated_pages/new_routing_checkbox_count/incorrect-answer.page";
+
+describe("Test routing using number of checkboxes checked", () => {
+  beforeEach(() => {
+    browser.openQuestionnaire("test_new_routing_checkbox_count.json");
+  });
+
+  it("Given a user selects 2 checkboxes, When they submit, Then they should be routed to the correct page", () => {
+    $(ToppingCheckboxPage.cheese()).click();
+    $(ToppingCheckboxPage.ham()).click();
+    $(ToppingCheckboxPage.submit()).click();
+
+    expect(browser.getUrl()).to.contain(CorrectAnswerPage.pageName);
+  });
+
+  it("Given a user selects no checkboxes, When they submit, Then they should be routed to the incorrect page", () => {
```

Comment:
Ah, I know why, because you are only checking the url, which would still be the same, but with the "Sorry, there is a problem with this service" on the page instead

Reply:
Added another transform "list_item_count" to fix this and show the number of the items selected instead of a list. I feel like this fits better with the test too.